lib/gui/models/supported-formats.js

```diff
@@ -131,6 +131,24 @@ SupportedFormats.service('SupportedFormatsModel', function() {
     return this.isSupportedImage(path.basename(image, `.${extension}`));
   };
 
+  /**
+   * @summary Check if an image seems a Windows image
+   * @function
+   * @public
+   *
+   * @param {String} image - image path
+   * @returns {Boolean} whether the image seems to be a Windows image
+   *
+   * @example
+   * if (SupportedFormatsModel.seemsWindowsImage('path/to/en_windows_7_ultimate_with_sp1_x86_dvd_u_677460.iso')) {
+   *   console.log('Looks like a Windows image');
+   * }
+   */
+  this.seemsWindowsImage = (image) => {
```

Comment:
I think it's a good idea to do it when we check for other images that require special treatment, but I'm not sure it's better to do it now. The advantage to make it now is that we already have some structure for adding new checks, the disadvantage is that it's harder to see the common patterns for different checks.

Anyways, let's also wait for @konmouz's input regarding the UX. BTW, I think it's also possible to change the `WarningModalService` to accept HTML, and have an anchor element in the displayed message, with an `onclick` event that will open the documentation in an external browser (by using [ngBindHtml](https://docs.angularjs.org/api/ng/directive/ngBindHtml) and [$sce.trustAsHtml](https://docs.angularjs.org/api/ng/service/$sce#trustAsHtml)) if you think it's a better UX experience.